aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1805,12 +1811,12 @@ def _call_exception_handler(self, exp: Exception, route: Route) -> Optional[Resp
         handler = self._lookup_exception_handler(type(exp))
         if handler:
             try:
-                return ResponseBuilder(handler(exp), route)
+                return self.response_builder_class(handler(exp), route)
             except ServiceError as service_error:
                 exp = service_error
 
         if isinstance(exp, ServiceError):
-            return ResponseBuilder(
+            return self.response_builder_class(
```

Comment:
same as before - is there a reason why we are allocating extra memory to have `ResponseBuilder` into a public attribute?